langgraph/graph/graph.py

```diff
@@ -235,9 +235,11 @@ def set_entry_point(self, key: str) -> None:
     def set_conditional_entry_point(
         self,
         path: Union[
-            Callable[..., str], Callable[..., Awaitable[str]], Runnable[Any, str]
+            Callable[..., Union[Hashable, list[Hashable]]],
```

Comment:
i think this now changes functionality to allow entry point to multiple, which i think is fine, but probably deserves a test?

Reply:
added a unittest!